rllib/utils/policy.py

```diff
@@ -185,13 +191,38 @@ def local_policy_inference(
         policy.agent_connectors
     ), "policy_inference only works with connector enabled policies."
 
+    # TODO(Artur): Remove this after we have migrated deepmind style preprocessing into
+    #  connectors (and don't auto-wrap in RW anymore)
+    if any(
```

Comment:
nit, nit, move this into a ``__util()`` function? so this occupies less real estate in the main body of the logics.

Reply:
Done thanks!